### DISCUSSION TITLE
Median spend and >1000 KIN

### DISCUSSION THREAD

**Discussion by ghost** [kinfoundation/kin-rfcs#111] [_ Archive - KRE 3.1 Discussion]
Breaking the ice here !!

What's the logic behind rewarding only spends above 1000 KIN?

It's like saying that an app with 100k AU spending 500 KIN each for a total of 50m KIN is not important enough to get rewarded!?!  Shouldn't this be the total amount of KIN spent in total (no matter the amount of each spend) rather than fixing it at >1000?

**Comment by wmougayar** [comment]
The previous number of 1 Kin was easily gameable and worthless at the same time. 
We are proposing 1000 Kin which is about 4 cents equivalent in order to encourage more meaningful use cases that have some value to an end users. 
In fact, the more successful Apps already have average user spends that are much higher than 1000 Kin, so that's the trend. 
If the value of 1000 Kin increases by a lot, one could always lower that threshold accordingly. 
If 1000 Kin isn't the right number, and 1 Kin neither, what number would you have proposed?

**Comment by ghost** [comment]
Well that's the point of this thread! Why having a fix number in the first place?

Why not use the total amount of KIN spent within an app and compare it to all the other apps in the ecosystem?

Whatever the number is, isn't it just weird to say that 1000 KIN will count as a valid spend but 950 wouldn't? At the end of the day, a spend is a spend.

Let's say we have app A and app B

App A total amount of KIN spent = 1m and they were all above 1000 KIN

App B total amount of KIN spent = 1m but half of them didn't count because they came from spends under 1000 KIN

Is this fair for app B?

**Comment by wmougayar** [comment]
The point is to increase the "value" and significance of these spends.
